packages/components/src/Layout/Semantics/Layout.tsx

```diff
@@ -46,16 +47,102 @@ export interface LayoutProps
 
 const hasAsideCSS = css`
   flex-direction: row;
-
   & > ${Section} {
     width: 0;
   }
 `
+export const Layout: FC<LayoutProps> = ({ children }) => (
+  <InnerLayout>{children}</InnerLayout>
+)
```

Comment:
The `InnerLayout` needs to "know" about `hasAside` & `fixed` to properly specify the CSS:

```suggestion
export const Layout: FC<LayoutProps> = ({ children, fixed, hasAside }) => (
  <InnerLayout hasAside={hasAside} fixed={fixed}>{children}</InnerLayout>
)
```